### PR TITLE
[MJAVADOC-762] don't share state between tests

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/test/java/org/apache/maven/plugins/javadoc/JavadocReportTest.java [hunk 1]
```diff
@@ -78,27 +79,37 @@ public class JavadocReportTest extends AbstractMojoTestCase {
...
     private File localRepo;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(JavadocReportTest.class);
 
-    /** {@inheritDoc} */
     @Override
     protected void setUp() throws Exception {
         super.setUp();
 
+        Path folder = Files.createTempDirectory("JavadocReportTest");
```
Comment (michael-o) [inline src/test/java/org/apache/maven/plugins/javadoc/JavadocReportTest.java:92]: directory, not folder

**Review by michael-o — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/maven/plugins/javadoc/JavadocReportTest.java (hunk 1 above)
Comment (elharo) [reply src/test/java/org/apache/maven/plugins/javadoc/JavadocReportTest.java]: done

**Review by michael-o — APPROVED** [review]
Good to merge after CI completion.

**Comment by jira-importer** [comment]
Resolve #1140

**Pull request merged**